xwiki-platform-core/xwiki-platform-search/xwiki-platform-search-solr/xwiki-platform-search-solr-api/src/main/java/org/xwiki/search/solr/internal/metadata/DefaultLinkStore.java

```diff
@@ -185,6 +190,12 @@ public Set<EntityReference> resolveBackLinkedEntities(EntityReference reference)
         return references;
     }
 
+    @Override
+    public ReadyIndicator waitReady()
```

Comment:
I'm wondering why one API uses `waitReady` and the other `getReadyIndicator` for apparently the same thing.

Reply:
Both APIs use `waitReady` now. If you have better suggestions please let me know.